### DISCUSSION TITLE
Airbyte OSS: Amazon Seller Partner API - New Finance API Endpoint stream

### DISCUSSION THREAD

**Discussion by nimeshvaghasiya** [airbytehq/airbyte#46944] [Connector Questions]
Can we have "/finances/2024-06-19/transactions" as a stream under current Amazon Selling Partner Source?
https://developer-docs.amazon.com/sp-api/docs/finances-api-v2024-06-19-reference#listtransactions

It has Standardized transaction information, and Marketplace level transactions, which is really needed one.

**Comment by intelc** [comment]
please!

[a comment by an automated account is not shown]
